### PR TITLE
Fix compilation to be compatible with IE11

### PR DESCRIPTION
**Pull request by ChrisDufourMB**
Fix issue where build would compile to ES6 code that is not compatible with IE11

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -13,10 +13,7 @@
     "declaration": true,
     "skipLibCheck": true,
     "stripInternal": true,
-    "lib": [
-      "es2017",
-      "dom"
-    ],
+    "lib": ["es2017", "dom"],
```
Comment (WillForrestMindBridge) [inline tsconfig.base.json:16]: Does it matter that the base config uses the es2017 lib, and the tree config references the es2018 lib?

**Review by WillForrestMindBridge — APPROVED** [review]
makes sense to me

**Review by shardyMBAI — APPROVED** [review]
[for reference](https://mindbridge-ai.atlassian.net/wiki/spaces/FG/pages/1488027671/Forking+a+Repository)

File: tsconfig.base.json (hunk 1 above)
Comment (ChrisDufourMB) [reply tsconfig.base.json:16]: I don't think it matters, I think what it does, in this case, is all modules are allowed to use es2017 code, and tree config can use es2018 code. Either way, both will be poly-filled down to es5.

**Pull request merged**